Flaky `TestBigQueryCaseInsensitiveMappingWithCache.testDropSchemaNameClash`

https://github.com/trinodb/trino/actions/runs/11337540050/job/31529331877
```
Error:  io.trino.plugin.bigquery.TestBigQueryCaseInsensitiveMappingWithCache.testDropSchemaNameClash -- Time elapsed: 2.159 s <<< FAILURE!
java.lang.AssertionError: Expected query to fail: DROP SCHEMA test_drop_case_sensitive_clash_9natu42rka [QueryId: 20241015_013916_00056_6na7g]
	at io.trino.testing.QueryAssertions.assertQueryFails(QueryAssertions.java:466)
	at io.trino.testing.AbstractTestQueryFramework.assertQueryFails(AbstractTestQueryFramework.java:445)
	at io.trino.plugin.bigquery.BaseBigQueryCaseInsensitiveMapping.testDropSchemaNameClash(BaseBigQueryCaseInsensitiveMapping.java:241)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.tryRemoveAndExec(ForkJoinPool.java:1499)
	at java.base/java.util.concurrent.ForkJoinPool.helpJoin(ForkJoinPool.java:2272)
	at java.base/java.util.concurrent.ForkJoinTask.awaitDone(ForkJoinTask.java:495)
	at java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:662)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.tryRemoveAndExec(ForkJoinPool.java:1499)
	at java.base/java.util.concurrent.ForkJoinPool.helpJoin(ForkJoinPool.java:2272)
	at java.base/java.util.concurrent.ForkJoinTask.awaitDone(ForkJoinTask.java:495)
	at java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:662)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1458)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2034)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:189)
```

--- a/plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMappingWithCache.java
+++ b/plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMappingWithCache.java
@@ -15,7 +15,11 @@ package io.trino.plugin.bigquery;
 
 import com.google.common.collect.ImmutableMap;
 import io.trino.testing.QueryRunner;
+import org.junit.jupiter.api.parallel.Execution;
 
+import static org.junit.jupiter.api.parallel.ExecutionMode.SAME_THREAD;
+
+@Execution(SAME_THREAD) // run single-threaded to avoid interference with other tests
 final class TestBigQueryCaseInsensitiveMappingWithCache
         extends BaseBigQueryCaseInsensitiveMapping
 {

